MIGRATION.md

```diff
@@ -0,0 +1,15 @@
+# Updating from Spotlight 1.x to 2.x
+
+Spotlight was updated to version 2 to ensure compatibility with version 8 of Sentry's JavaScript SDKs.
+
+Here's what you need to do to update to version 2:
+
+- Ensure that you use a version 8 of your `@sentry/<sdk>` SDKs
```

Comment:
I would propose to support v7.99.0+ (https://github.com/getsentry/sentry-javascript/blob/v7/CHANGELOG.md#7990) which should work, and makes it easier for people to update. That version should have stuff that we need to make this work, I believe?

Reply:
we could probably even go down to 7.96.0 (https://github.com/getsentry/sentry-javascript/blob/v7/CHANGELOG.md#7960) where we started exposing the functional integrations in browser. 

Or we set it to 7.100.0 where we exposed the integrations in node. Probably the safer choice just to be sure the API is available everywhere.

Reply:
> we could probably even go down to 7.96.0
oh nvm, 7.97.0 it is (missed that the 7.96.0 release was incomplete)

Reply:
update: 7.97 doesn't yet support `afterAllSetup`. 7.99.0 does, so trying this next

Reply:
ok, we now require 7.99.0 or greater 👍